src/CQRS/LeanCode.CQRS.AspNetCore/Middleware/CQRSMiddleware.cs

```diff
@@ -1,23 +1,29 @@
 using System.Diagnostics.CodeAnalysis;
 using LeanCode.CQRS.AspNetCore.Serialization;
 using LeanCode.CQRS.Execution;
+using LeanCode.Logging;
 using Microsoft.AspNetCore.Http;
-using Serilog;
 
 namespace LeanCode.CQRS.AspNetCore.Middleware;
 
 public class CQRSMiddleware
 {
     private static readonly byte[] NullString = "null"u8.ToArray();
 
-    private readonly ILogger logger = Log.ForContext<CQRSMiddleware>();
+    private readonly ILogger<CQRSMiddleware> logger;
 
     private readonly CQRSMetrics metrics;
     private readonly ISerializer serializer;
     private readonly RequestDelegate next;
 
-    public CQRSMiddleware(CQRSMetrics metrics, ISerializer serializer, RequestDelegate next)
+    public CQRSMiddleware(
```

Comment:
One last (not priority, not mandatory) request - let's put the logger as a last parameter (I think this is also a convention in the .NET examples, but might be wrong here).